app/views/accounts/_connected_app.html.erb

```diff
@@ -1,26 +1,21 @@
 <div class="padding-1 border-top border-left border-right border-primary-light">
-  <div class="grid-row grid-gap-2">
-    <div class="grid-col-fill">
-      <div class="text-bold truncate">
-        <% if identity.return_to_sp_url.present? %>
-          <%= link_to(identity.display_name, identity.return_to_sp_url) %>
-        <% else %>
-          <%= identity.display_name %>
-        <% end %>
-      </div>
-
-      <%= t(
-            'account.connected_apps.associated',
-            timestamp: render(TimeComponent.new(time: identity.created_at)),
-          ).html_safe %>
-    </div>
-    <% if identity.service_provider_id %>
```

Comment:
I can't imagine a scenario where this would be falsey? I guess we'll find out if there's regression coverage.

Reply:
Looks like this was added in #3747, I'll check to see what needs to be done here and if we can add some regression coverage.

Reply:
Thankfully it looks like there's already a [failing spec](https://github.com/18F/identity-idp/blob/ac4b9e5aa451d2b8fde3938b2c4ac56b66ce2800/spec/views/accounts/connected_accounts/show.html.erb_spec.rb) here.

Reply:
Restored the original condition in f9010ac2b1.